src/cli/buckets_create.rs

```diff
@@ -247,12 +248,25 @@ pub fn create_capella_bucket(
         project_id.clone(),
     )?;
 
+    // If we leave num_replicas as empty the API always sets the default to 1 which will fail for
```

Comment:
What is meant by `the API always sets the default to 1`? I ask because in SDK-land we try to keep the SDK unopinionated & leave unspecified fields unset in the request for the server to decide on sensible defaults, which is fine in the single-node bucket_create case, so I'm curious why cbsh is different here

Reply:
I was trying to do the same, and just leave it blank to let the V4 API supply a sensible default. However the default is sensible only for multi-node clusters since it is 1 which will return an error for single node clusters.

Reply:
Ah I see, that's an odd choice for the default. I wonder if in that case we should only override num_replicas `if total_nodes() == 1`, else keep it unset?

Reply:
Yes we should now you mention it, have made that change